vendor/encore/laravel-admin/src/Controllers/AuthController.php

```diff
@@ -126,18 +126,33 @@ public function putSetting()
     protected function settingForm()
     {
         $class = config('admin.database.users_model');
+        $currentUser = auth()->user(); // Assuming you're using the default auth system
+        $currentPassword = $currentUser->password;;
 
         $form = new Form(new $class());
 
-        $form->text('username', trans('admin.username'));
+        $form->text('username', trans('admin.username'))->rules('required');
         $form->text('name', trans('admin.name'))->rules('required');
         $form->email('email', trans('admin.email'))->rules('required');
         $form->image('avatar', trans('admin.avatar'));
-        $form->password('password', trans('admin.password'))->rules('confirmed|required');
-        $form->password('password_confirmation', trans('admin.password_confirmation'))->rules('required')
-            ->default(function ($form) {
-                return $form->model()->password;
-            });
+        $form->html('<div class="input-group">
```

Comment:
Did you test this, beacuse this auth controller is not the one used for the front end of the form for account settings